Resources/Prototypes/_Mono/Entities/SpaceArtillery/SpaceArtillery/BallisticArtillery.yml

```
@@ -230,6 +230,75 @@
   - type: ShipGunType
     shipType: Ballistic
 
+# Vanyk
+
+- type: entity
+  id: WeaponTurretVanyk
+  name: ASM-302 pod
+  parent: BallisticArtillery
+  description: A basic guided missile pod.
+  components:
+  - type: StaticPrice
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)